### DISCUSSION TITLE
Webdriver IO with JDBC connection with SQL database

### DISCUSSION THREAD

**Discussion by danferns37** [webdriverio/webdriverio#10445] [General]
Can anyone share a working example of Webdriver IO with JDBC connection with SQL database ,I need to connect to database for test data

**Comment by christian-bromann** [comment]
@danferns37 you might want to use something like this: https://www.npmjs.com/package/node-jdbc-driver. Fyi: there is nothing specific to WebdriverIO on this. Any example that shows how to make a JDBC connection in Node.js will help you 😉

**Reply by danferns37** [reply]
Thanks

**Reply by danferns37** [reply]
Getting a error cannot find module appium ,tried installing through npm I appium in wdio project still get the same error
